ports/stm32/boards/NUCLEO_F767ZI/mpconfigboard.h

```diff
@@ -71,6 +72,11 @@ void NUCLEO_F767ZI_board_early_init(void);
 #define MICROPY_HW_USB_VBUS_DETECT_PIN (pin_A9)
 #define MICROPY_HW_USB_OTG_ID_PIN      (pin_A10)
 
+// SD card detect switch
+#define MICROPY_HW_SDCARD_DETECT_PIN        (pin_G2)
+#define MICROPY_HW_SDCARD_DETECT_PULL       (GPIO_PULLUP)
+#define MICROPY_HW_SDCARD_DETECT_PRESENT    (GPIO_PIN_RESET)
```

Comment:
If a user does modify the board to add an SD card they might use a different pin than PG2.

Reply:
Well yes and no: the nucleo_H743zi is the same board also w/o a sdcard. So yes leave the definition in, but set it to 0 if you prefer. And for consistency disable it too in the other port :-). By the way the pins are nicely positioned next to each other and labeled by ST on the board. Remember the small board in your mail... the difference there is the inversion of the signal...

Reply:
Ah, I didn't appreciate that it was enabled on the H743ZI even though that also doesn't have an SD card (see cf1d6df05acc22b641fb97de6a87b100b4fcfcb1)

Ok, if the pins are labelled by ST then it makes sense to enable it so it can be easily used.  Maybe good to add a comment here about the detect switch being a convenience and might need modifying for a particular use.